# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../busybox/newer.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./archival/libunarchive/data_extract_all.c.orig	2009-11-23 13:57:45.932006498 -0500
+++ ./archival/libunarchive/data_extract_all.c	2009-11-23 14:31:42.520002366 -0500
@@ -37,7 +37,7 @@
 				bb_perror_msg_and_die("cannot stat old file");
 			}
 		}
-		else if (statbuf.st_mtime <= file_header->mtime) {
+		else if (statbuf.st_mtime >= file_header->mtime) {
 			if (!(archive_handle->ah_flags & ARCHIVE_EXTRACT_QUIET)) {
 				bb_error_msg("%s not created: newer or "
 					"same age file exists", file_header->name);
